WebExample/tests/styles.spec.ts

```diff
@@ -0,0 +1,66 @@
+import {test, expect} from '@playwright/test';
+import type {Page} from '@playwright/test';
+import * as TEST_CONST from '../../testConstants';
+import {setupInput} from './utils';
+
+const testMarkdownContentStyle = async ({styleName, style, page}: {styleName: string; style: string; page: Page}) => {
+  const inputLocator = await setupInput(page);
+
+  const elementHandle = inputLocator.locator('span', {hasText: styleName}).last();
+
+  let elementStyle;
+
+  if (elementHandle) {
+    await elementHandle.waitFor({state: 'attached'});
+
+    elementStyle = await elementHandle.getAttribute('style');
+  }
```

Comment:
Does it make sense to move it to an utility function? Looks like we use it again in `paste content` test.

Reply:
k, I'll move it